Backpack/src/main/java/net/skyscanner/backpack/calendar/BpkCalendar.kt

```diff
@@ -39,6 +40,14 @@ open class BpkCalendar @JvmOverloads constructor(
     updateYearPill(controller.startDate.year)
   }
 
+  fun setOnScrollListener(listener: AbsListView.OnScrollListener) {
```

Comment:
We we follow my suggestion bellow we can change this to `addScrollListener` and also add a new `removeScrollListener`